packages/report/src/dashboard/components/urlsWithIssues/index.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useMemo, useState } from 'react';
+import { Resizable } from 're-resizable';
+import { noop, type ErroredOutUrlsData } from '@google-psat/common';
+import { I18n } from '@google-psat/i18n';
+import {
+  Table,
+  TableProvider,
+  type TableColumn,
+  type InfoType,
+  type TableRow,
+  type TableFilter,
+} from '@google-psat/design-system';
+
+interface ErroredOutUrlsProps {
+  erroredOutUrls: ErroredOutUrlsData[];
+}
+
+const ErroredOutUrls = ({ erroredOutUrls }: ErroredOutUrlsProps) => {
+  const [selectedRow, setSelectedRow] = useState<ErroredOutUrlsData>();
+
+  const tableColumns = useMemo<TableColumn[]>(
+    () => [
+      {
+        header: 'URL',
+        accessorKey: 'url',
+        cell: (info: InfoType) => info,
+        enableHiding: false,
+      },
+      {
+        header: 'Error Description',
+        accessorKey: 'errorMessage',
+        cell: (info: InfoType) => info,
+      },
+      {
+        header: 'Error Code',
+        accessorKey: 'errorCode',
+        cell: (info: InfoType) => (
+          <span className="w-full flex justify-center">{info}</span>
+        ),
+      },
+    ],
+    []
+  );
+
+  const filters = useMemo<TableFilter>(() => ({}), []);
+
+  const searchKeys = useMemo<string[]>(() => ['url'], []);
```

Comment:
What's the purpose of using `useMemo` here?

Reply:
It was being used in other places as well. Will change that.